<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWin</class>
 <widget class="QMainWindow" name="CMainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>data server notifyer</string>
  </property>
  <widget class="QWidget" name="m_iCentralWidget">
   <property name="maximumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QWidget" name="widget" native="true">
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>1221</width>
           <height>391</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="title">
          <string>Table Config and Click to Notify</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>81</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>TableA</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="tableA_videoID">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>60</y>
            <width>151</width>
            <height>32</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>BG01</string>
          </property>
         </widget>
         <widget class="QLabel" name="lable_2">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>61</y>
            <width>71</width>
            <height>41</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>videoID:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>100</y>
            <width>80</width>
            <height>32</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>obsID:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="tableA_obsID">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>100</y>
            <width>121</width>
            <height>32</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>bgtz-a</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>140</y>
            <width>80</width>
            <height>32</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>tableID:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>30</y>
            <width>101</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Data Server</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>70</y>
            <width>81</width>
            <height>32</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>host or IP:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="datasvr_host">
          <property name="geometry">
           <rect>
            <x>460</x>
            <y>70</y>
            <width>161</width>
            <height>32</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>127.0.0.1</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>380</x>
            <y>110</y>
            <width>61</width>
            <height>32</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>port:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="datasvr_port">
          <property name="geometry">
           <rect>
            <x>460</x>
            <y>110</y>
            <width>120</width>
            <height>32</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>17010</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>730</x>
            <y>150</y>
            <width>81</width>
            <height>32</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>tableID:</string>
          </property>
         </widget>
         <widget class="QToolButton" name="Btn_SwitchToTableA">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>250</y>
            <width>260</width>
            <height>64</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>260</width>
            <height>64</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>64</height>
           </size>
          </property>
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>120</green>
                <blue>215</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>240</red>
                <green>240</green>
                <blue>240</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>240</red>
                <green>240</green>
                <blue>240</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Switch To TableA</string>
          </property>
         </widget>
         <widget class="QToolButton" name="Btn_SwitchToTableB">
          <property name="geometry">
           <rect>
            <x>690</x>
            <y>250</y>
            <width>260</width>
            <height>64</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>260</width>
            <height>64</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>64</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Switch To TableB</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>710</x>
            <y>30</y>
            <width>81</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>TableB</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>730</x>
            <y>70</y>
            <width>71</width>
            <height>32</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>videoID:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="tableB_videoID">
          <property name="geometry">
           <rect>
            <x>800</x>
            <y>70</y>
            <width>151</width>
            <height>32</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>BG02</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="tableA_tableID">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>140</y>
            <width>61</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>730</x>
            <y>110</y>
            <width>61</width>
            <height>32</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>obsID:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="tableB_obsID">
          <property name="geometry">
           <rect>
            <x>800</x>
            <y>110</y>
            <width>120</width>
            <height>32</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>bgtz-b</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="tableB_tableID">
          <property name="geometry">
           <rect>
            <x>800</x>
            <y>150</y>
            <width>61</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
